packages/dashboard/src/contexts/DashContext/state.ts

```diff
@@ -49,6 +49,7 @@ export const initialState: State = {
   }
 };
 
+//To-do: set-analytics in reducer
```

Comment:
Is this comment from the dashboard work? :)

Reply:
eee... yes they sneaked in... Will remove.